### PR TITLE
Added debug option to force refreshing of tokens

### PR DESCRIPTION
**Pull request by aliak00**

### CODE REVIEW

File: Source/Manager/TaskManager/TaskManager.swift [hunk 1]
```diff
@@ -77,6 +77,21 @@ class TaskManager {
                         DispatchQueue.main.async {
                             completion?(result)
                         }
+
+                        if SDKConfiguration.shared.invalidateteAuthTokenAfterSuccessfullRequest {
```
Comment (minarikg) [inline Source/Manager/TaskManager/TaskManager.swift]: I would protect this part with #if DEBUG, just to be sure

File: Source/Manager/SDKConfiguration/SDKConfiguration.swift [hunk 2]
```diff
@@ -36,4 +36,16 @@ public class SDKConfiguration {
...
     }
+
+    /**
+     Set this to debug access token and refreshing requests. If set to true then every successful request
+     will also invalidate the access token so that a refresh will be forces
+
+     Only settable in debug mode
+     */
+    #if DEBUG
+        public var invalidateteAuthTokenAfterSuccessfullRequest = false
```
Comment (Torsph) [inline Source/Manager/SDKConfiguration/SDKConfiguration.swift]: Why cannot this be set to `true` when its set to `true` in `Initialize.init` under debug?

File: Source/Manager/TaskManager/TaskManager.swift [hunk 3]
```diff
@@ -77,6 +77,23 @@ class TaskManager {
                         DispatchQueue.main.async {
                             completion?(result)
                         }
+
+                        #if DEBUG
+                        if SDKConfiguration.shared.invalidateteAuthTokenAfterSuccessfullRequest {
```
Comment (Torsph) [inline Source/Manager/TaskManager/TaskManager.swift]: Should this be inside its own function to separate it more from the main code?

File: Source/Manager/SDKConfiguration/SDKConfiguration.swift (hunk 2 above)
Comment (aliak00) [reply Source/Manager/SDKConfiguration/SDKConfiguration.swift]: Good question. My thought was that then ze clients will get the behavior by default when they are developing and I thought opt-in is nicer for debug stuff.

But ... 🤷‍♂️

File: Source/Manager/TaskManager/TaskManager.swift (hunk 3 above)
Comment (aliak00) [reply Source/Manager/TaskManager/TaskManager.swift]: 🗡 😛

**Pull request merged**